packages/react-scripts/package.json

```diff
@@ -21,11 +21,11 @@
     "react-scripts": "./bin/react-scripts.js"
   },
   "dependencies": {
-    "@babel/core": "7.0.0-beta.38",
-    "@babel/runtime": "7.0.0-beta.38",
+    "@babel/core": "7.0.0-beta.42",
+    "@babel/runtime": "7.0.0-beta.42",
```

Comment:
can you git rebase your changes on top of the next branch? babel is already upgraded to 42 in next and it will make the PR easier to review.

Reply:
Will do